### PR TITLE
FET-621 Fix tooltip and FET-620 fix data label

### PR DESCRIPTION
**Pull request by tomassatinsky**
---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

File: libs/sdk-ui-charts/styles/scss/tooltip.scss [hunk 1]
```diff
@@ -152,7 +152,9 @@
         visibility: hidden;
     }
 
-    z-index: 3005;
+    // !important is necessary because z-index have been added to the element by Highcharts in
+    // https://github.com/highcharts/highcharts/commit/48a82ce80ba946839e3a1280e63ca02ab1f88ec0
+    z-index: 3005 !important;
```
Comment (tomassatinsky) [inline libs/sdk-ui-charts/styles/scss/tooltip.scss:157]: This is the only change of FET-621

**Comment by tomassatinsky** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2587/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4219/ : FAILURE in 7m 35s

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/693/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2588/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/693/ : UNSTABLE in 15m 21s

**Comment by tomassatinsky** [comment]
extended test

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/695/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2595/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/695/ : UNSTABLE in 15m 19s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by tomassatinsky** [comment]
extended test

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/696/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2600/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/696/ : UNSTABLE in 36m 31s

**Comment by no23reason** [comment]
The failing tests are unrelated to the changes here and are most likely flaky.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1073

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/783/ : FAILURE in 2m 54s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1046/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4236/ : FAILURE in 26m 17s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-el7/146/ : SUCCESS in 11m 37s (non-voting)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2609/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4241/ : SUCCESS in 15m 34s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- gooddata-ui-sdk-fossa-scanning-zuul-el7 : SKIPPED (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1049/ : FAILURE in 19s
- gooddata-ui-sdk-unit-tests : SKIPPED
- gooddata-ui-sdk-sonarqube-scan-zuul-el7 : SKIPPED (non-voting)

**Review by xMort — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1073

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/786/ : FAILURE in 2m 57s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1050/ : SUCCESS in 2m 18s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4244/ : SUCCESS in 16m 33s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-el7/149/ : SUCCESS in 14m 22s (non-voting)

**Pull request merged**